R/readEDGE.R

```diff
@@ -17,7 +17,7 @@ readEDGE <- function(subtype = c("FE_stationary", "FE_buildings", "Capital", "Ca
   subtype <- match.arg(subtype)
 
   # input data version
-  ver <- "1.05"
+  ver <- "1.06"
```

Comment:
is a different data source loaded by this change?

Reply:
Yes, it loads the new EDGE-B demand trajectories. `ver` defines the subfolder in `inputdata/sources/EDGE/` from which the data is read.